### PR TITLE
Add Dockerfile for production

### PR DESCRIPTION
**Pull request by MattBlack85**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+TAG = latest
+
+docker-build:
+	@docker build . -t meetpy:$(TAG)
+
+docker-login:
+	@echo $DOCKERHUB_PASSWORD | docker login -u $DOCKERHUB_USER --password-stdin
+
+docker-push: docker-login
+	@docker push /meetpy:$(TAG)
```
Comment (MattBlack85) [inline Makefile]: TBD

File: requirements/base.txt [hunk 2]
```diff
@@ -22,3 +22,4 @@ six==1.10.0
 pytest==3.6.0
 pytest-cov==2.5.1
 pytest-django==3.2.1
+supervisor==4.0.4
```
Comment (wmalinowski) [inline requirements/base.txt]: Shouldn't this go to the production.txt?

Comment (MattBlack85) [reply requirements/base.txt]: dropped supervisor

**Comment by artcz** [comment]
```
 ---> b4886e34b5ad
Step 9/14 : ADD meetpy/ /opt/meetpy/meetpy
 ---> e3b56bf2560a
Step 10/14 : COPY ./deploy/supervisord/meetpy.ini /etc/supervisor/supervisord.d/meetpy.ini
COPY failed: stat /var/lib/docker/tmp/docker-builder174571210/deploy/supervisord/meetpy.ini: no such file or directory
Makefile:8: recipe for target 'docker-build' failed

```

File: Dockerfile-dev [hunk 3]
```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
```
Comment (gbdlin) [inline Dockerfile-dev]: Should be based on prod dockerfile

File: Dockerfile-dev [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+WORKDIR /usr/src/app
+COPY requirements/ /usr/src/app/requirements/
+
+RUN pip install --no-cache-dir --requirement requirements/production.txt
```
Comment (gbdlin) [inline Dockerfile-dev]: Doesn't look like dev requirements ;)

Comment (MattBlack85) [reply Dockerfile-dev]: made consistent with Dockerfile and using `requirements/base.txt` now

File: docker-compose-dev.yml [hunk 5]
```diff
@@ -1,15 +1,16 @@
...
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
     volumes:
       - ./:/usr/src/app
     working_dir: /usr/src/app/meetpy
     ports:
-      - "8080:8000"
+      - "18000:8000"
```
Comment (ivellios) [inline docker-compose-dev.yml]: 8000 is typical for django, so for devs I would leave it as 8000.

**Pull request merged**